### DISCUSSION TITLE
412 Gmail_API: Request had insufficient authentication scopes.

### DISCUSSION THREAD

**Discussion by Jamesishaku** [emailjs-com/help#31] [Q&A] [answered]
I am using NextJs and emailjs to submit a form to my Gmail but I kept getting this error message "412Gmail_API: Request had insufficient authentication scopes."

Please, how can I fix it?

**Comment by xr0master** [comment, marked as answer]
To solve this issue, you need to go to the EmailJS dashboard -> Services, click on the Gmail service in your list, and reconnect your account (click on the Edit button, disconnect, and then connect again).
On the popped-up screen, please press the **"Allow"** button to allow the service to send the emails through your Gmail account.

**Comment by Jamesishaku** [comment]
Disconnecting and reconnecting worked. Thank you.
